turbo.json

```diff
@@ -22,6 +14,11 @@
     "test:watch": {
       "cache": false,
       "persistent": true
+    },
+    "typecheck": {},
+    "//#format-and-lint": {},
+    "//#format-and-lint:fix": {
```

Comment:
What's with this weird syntax `//#`? I see it in https://turbo.build/repo/docs/guides/tools/biome#create-a-root-task but I'm not sure why it is like that.

Reply:
I think so that `pnpm exec turbo format-and-lint` works, but on my machine 

- 2.7s for pnpm exec turbo format-and-lint (cached)
- 0.3s for  time pnpm format-and-lint

So I will remove it as running biome directly is preferred.